.gitignore

```diff
@@ -1,3 +1,6 @@
 /builddir/
 /repo/
 /.flatpak-builder/
+update
```

Comment:
I meant this and the following now two lines.

Reply:
If you like I can fix it as well.

Reply:
Fixed.